platform/linux-generic/odp_packet_io.c

```diff
@@ -1830,6 +1830,10 @@ int odp_pktio_capability(odp_pktio_t pktio, odp_pktio_capability_t *capa)
 		capa->vector.min_tmo_ns = 0;
 	}
 
+	capa->reassembly.ip = ODP_SUPPORT_NO;
+	capa->reassembly.ipv4 = ODP_SUPPORT_NO;
+	capa->reassembly.ipv6 = ODP_SUPPORT_NO;
```

Comment:
These are not of type odp_support_t but odp_bool_t.

Reply:
Agreed. Will address this in v18.